client/src/components/Overview.jsx

```diff
@@ -11,10 +11,13 @@ class Overview extends React.Component {
     super(props);
     this.state = {
       restaurant: {},
+      displayAll: false,
     };
+    this.getIntialData();
```

Comment:
Is getIntialData supposed to be spelled like that instead of getInitialData? Just making sure it's not a typo that could break something.

Reply:
oops Good catch! Thanks! :D